ui/src/pages/Cluster/New/Observer.tsx

```diff
@@ -173,7 +192,21 @@ export default function Observer({ storageClasses, form }: ObserverProps) {
           </Col>
         </Row>
         <p className={styles.titleText}>storage</p>
-
+        <p className={styles.subTitleText}>存储卷配置</p>
```

Comment:
Why not use `CustomFormItem`? I don't think this form item for pvc protection is special among other items.

Reply:
Style does not match

Reply:
Style does not match